src/core/messages.ts

```diff
@@ -232,10 +249,26 @@ export interface Messages extends EmitsDiscontinuities {
    *
    * @returns A promise that resolves when the message was deleted.
    * @param message - The message to delete.
-   * @param deleteMessageParams - The optional parameters for deleting the message.
+   * @param deleteMessageParams - Optional details to record about the delete action.
    * @return A promise that resolves to the deleted message.
    */
-  delete(message: Message, deleteMessageParams?: DeleteMessageParams): Promise<Message>;
+  delete(message: Message, deleteMessageParams?: ActionDetails): Promise<Message>;
```

Comment:
Okay, I promise this is my last comment! Should be DeleteMessageParams no ActionDetails now